src/screens/ExplainSheet.js

```diff
@@ -461,6 +465,19 @@ export const explainers = (params, theme) => {
       text: AVALANCHE_EXPLAINER,
       title: lang.t('explain.avalanche.title'),
     },
+    degen: {
+      emoji: '⛽️',
+      extraHeight: 144,
+      logo: <ChainBadge network={networkTypes.degen} marginBottom={8} position="relative" size="large" />,
+      readMoreLink: buildRainbowLearnUrl({
+        url: 'https://learn.rainbow.me/a-beginners-guide-to-layer-2-networks',
```

Comment:
looks like we're gonna need to make a new learn doc

Reply:
yeah, can ping support to atleast add a ref for degen